comiccrawler/grabber.py

```diff
@@ -15,7 +15,8 @@
 from .profile import get as profile
 
 default_header = {
-	"User-Agent": "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:34.0) Gecko/20100101 Firefox/34.0",
+	"User-Agent": "Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36
+",
```

Comment:
Syntax error.

Reply:
when I edit in github web directly  it web into two line automatically, dunno how to solve it, sorry about that.